src/outpack/util.py

```diff
@@ -100,3 +100,10 @@ def relative_path_array(files, name):
             msg = f"Expected {name} path '{f}' to be a relative path"
             raise Exception(msg)
     return files
+
+
+def read_string(path):
```

Comment:
surprised we don't have this already!

I think you can also do this with:

```
with open(path) as f:
  return f.read().rstrip()
```

The rstrip part is probably worth changing to though, which breaks your test, but I think the main reason we use this in the R port is to allow for a trailing newline or not